### DISCUSSION TITLE
Horizontal navbar?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#694] [Q&A] [answered]
Is there are a way to use `<Header>` component to make a horizontal navbar like the one on Github site? Meaning, there would be navigational links in `<Header>` and when the screen is too narrow they will be hidden under `<Burger>` menu.

As of now, you can use `<Header>` without `<AppShell>`, but the links inside `<Header>` will just disappear on breakpoint and the burger menu will be empty - since as far as I understand `<Header>` is tied to (vertical left side) `<Navbar>` of `<AppShell>` to put the links into burger menu? We can't control what's being put in the burger menu, it's always looking for `<Navbar>` contents?

But I don't need the left-side `<Navbar>`, I want all links to be in the top `<Header>` (like on Github site) and to be properly hidden inside the burger menu on mobile.

How to achieve this with Mantine?

**Comment by rtivital** [comment, marked as answer]
See example – https://codesandbox.io/s/intelligent-babycat-mlwm0?file=/src/App.tsx

**Reply by danmaz74** [reply]
Thank you for the example!

I just started using mantine and not having a default horizontal navbar is the biggest missing feature I found so far. Would it make sense to open an issue asking for it?

**Reply by rtivital** [reply]
Sandbox shows how to implement it. What else do you need from the library?

**Reply by danmaz74** [reply]
I personally will just implement it this way, but my suggestion would be to add it as a documented configuration choice for AppShell and/or Navbar, as it looks to me like a very common use case. Something like

`navbarDirection='horizontal'`

**Reply by danmaz74** [reply]
Or in alternative - having a HorizontalNavbar that you can include inside header

**Reply by rtivital** [reply]
`navbarDirection='horizontal'` is `Header` component

**Reply by danmaz74** [reply]
OK, I just discovered that what I was really looking for exists in https://ui.mantine.dev/category/headers - my confusion came from the fact that I didn't' search on ui.mantine.dev, but only on the (main?) mantine docs. My bad.
